src/js/src/components/ngrams/chartsCore/chartEngines/LineChart.js

```diff
@@ -0,0 +1,28 @@
+import { Line, mixins } from 'vue-chartjs'
+import SearchHelper from '../../searchHelper'
+const { reactiveProp } = mixins
+
+export default {
+  extends: Line,
+  mixins: [reactiveProp, SearchHelper],
+  props: {
+    chartData: {
+      type: Object,
+      default: null
+    },
+    options: {
+      type: Object,
+      default: null
+    }
+  },
+  mounted () {
+    // this.chartData is created in the mixin.
+    // If you want to pass options please create a local options object
+    this.renderChart(this.chartData,  this.options)
```

Comment:
This seems funky to me somehow. You never really create a new chart (like let chart = new Chart()) - but it somehow works with this vue-chartjs magic. Very funky, imho

Reply:
Very valid point and one of the drawbacks of using wrappers:(

I think we perhaps should have a future discussion about the usage of wrappers vs 'the raw product code'. 

The logic here is that the LineChart component is an instance of ChartJs hence the _this.renderChart_

Reply:
But isn't one of the drawbacks here, that you can't get the chart object (which you seem to need in your onClick function that's causing you problems?

Reply:
I have access to the chart object here in LineChart no problem (also stated in the docs) but if I tamper with _this.option_ here even as data/computed property(!!) all hell breaks loose on first search. After first search all is fine.

Reply:
Hmmm. It sounds... intriguing! I hope you find your solution to this at some point :pray: